### ISSUE TITLE
vmboxmanage is case sensitive on my Mac so fin doesn't install

### ISSUE THREAD

**Issue by DigitalFrontiersMedia** [docksal/docksal#42]
Trying to install fin on a 2011 MacBook Pro with OS X El Capitan (10.11.4 (15E65)) results in error message saying my version of VirtualBox doesn't meet the requirements.  Even though it's the latest version from the URL provided in your installation steps.

In my case, fin is trying to call "vboxmanage" which doesn't exist.  "VBoxManage" does, however.

I don't know if this is a case of my specific disk configuration or if this is a difference in how fin should react between Windows, Mac, Linux type of problem.  But I had to edit line 46 to use the proper capitalization for my machine.  After doing so, fin installs as expected.

**Comment by lmakarov** [comment]
@DigitalFrontiersMedia looks like you have you system partition formatted as case-sensitive. By default OS X formats it's system partition as not case-sensitive.

```
$ diskutil info / | grep "File System Personality"
   File System Personality:  Journaled HFS+
```

I presume in you case the output will look like:
```
File System Personality:  Case-sensitive Journaled HFS+
```
Can you please confirm?

**Comment by DigitalFrontiersMedia** [comment]
Oh, yes, I know it's case sensitive for a fact.  And I can confirm that's how it reads:
`MotherPhoenix-3:drupal7 stephen$ diskutil info / | grep "File System Personality"
   File System Personality:  Case-sensitive Journaled HFS+
MotherPhoenix-3:drupal7 stephen$`

Is it a problem that my drive is formatted in this way?  If the docksal code were written for case-sensitive, then it would run no matter how someone formatted their drive, yes?

**Comment by achekulaev** [comment]
Fixed in 0.10.1 but I would not recommend using case sensitive FS. It can create issues with different software

**Comment by DigitalFrontiersMedia** [comment]
> I would not recommend using case sensitive FS. It can create issues with different software

Understood.  Thanks for the advice.
